pingcap/tiflash
# Add pending duration metrics of I/O rate limiter.

### What problem does this PR solve?

Issue Number: close #7598

### What is changed and how it works?
Add metrics of I/O pending duration in `RateLimiter`.

### Check List

Tests 

  - Set config: `storage.io_rate_limit.max_bytes_per_sec: 10485760`
  - Run tests:
    - tiup bench ch --warehouses 1500 -T 0 -t 1 run -H 10.2.12.81 -P 8244 -D chbenchmark
    - tiup bench ch --warehouses 1500 -T 10 -t 0 --time 10h run -H 10.2.12.81 -P 8244 -D chbenchmark
  - Check metrics in Grafana:
![image](https://github.com/pingcap/tiflash/assets/6143402/be5db026-8f21-4589-9deb-49a4c70e77c4)
![image](https://github.com/pingcap/tiflash/assets/6143402/50ed2514-176c-4457-b4bd-356bb1b29e97)




Side effects


Documentation


### Release note



```release-note
None
```
